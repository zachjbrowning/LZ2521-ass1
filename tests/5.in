s a
s b
s c
l
l
s ajouter
s r
s re
s ra
s rec
l
