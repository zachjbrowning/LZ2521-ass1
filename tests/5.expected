> ajouter to add
aller to go
appeler to call
arriver to arrive
attendre to wait
avoir to have
> > commencer to start
comprendre to understand
> 1 ajouter to add
1 aller to go
1 appeler to call
1 arriver to arrive
1 attendre to wait
1 avoir to have
1 commencer to start
1 comprendre to understand
> 1 ajouter to add
1 aller to go
1 appeler to call
1 arriver to arrive
1 attendre to wait
1 avoir to have
1 commencer to start
1 comprendre to understand
> ajouter to add
> rappeler to remind
recevoir to receive
rendre to return
> recevoir to receive
rendre to return
> rappeler to remind
> recevoir to receive
> 3 recevoir to receive
2 ajouter to add
2 rappeler to remind
2 rendre to return
1 aller to go
1 appeler to call
1 arriver to arrive
1 attendre to wait
1 avoir to have
1 commencer to start
> 